psi4/src/psi4/libmints/molecule.cc

```diff
@@ -424,7 +426,7 @@ Vector3 Molecule::center_of_mass() const {
     total_m = 0.0;
 
     for (int i = 0; i < natom(); ++i) {
-        double m = mass(i);
+        double m = mass(i, false);
```

Comment:
This one is probably going to be `true` most of the time, but you may want to have this function take an optional for explicit inclusion of ghost atoms.